### ISSUE TITLE
[Accessibility] Audio cues stopped working in Chrome

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#178935]
CC @meganrogge and @joaomoreno

Audio cues stopped working in Chrome again. There might be an inadvertent regression.

## Reproducible Steps

1. In Chrome, go to https://github.dev/microsoft/vscode

1. In command palette, run `Help: List Audio Cues`

## System Info

Version: 1.78.0-insider
Commit: f31dffc7f9b93a1cdef21243ec30dea7d489a615
Date: 2023-03-31T01:37:26.501Z
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36

**Comment by meganrogge** [comment]
In dev tools
```
       Refused to load media from 'https://assets.github.dev/insider/2da5a0024fe2ca407f741f19340f1a10f4d692af/out/vs/platform/audioCues/browser/media/warning.mp3' because it violates the following Content Security Policy directive: "media-src https://main.vscode-cdn.net/insider/
            https://main.vscode-cdn.net/stable/".
```
